README.md

```diff
@@ -143,12 +147,6 @@ Multiple main classes detected, select one to run:
 If you pick the program importer, it will import everything which is the same as explicitly passing in `Int.MaxValue`.
 
 
-### Enumerated Types
-
-Enumerated types are represented by tables named `e_whatever` which are the source for generated code on the Scala side. After compiling if you look in `modules/core/target/scala-2.11/src_managed/gem` you will see the source. The rationale for this is that the database becomes the source of truth, which makes things like filter wavelengths, etc., available for querying and reporting.
```

Comment:
This still seems like useful information.

Reply:
I'll add it back. It kind of duplicates some information above.